gravity.sh

```diff
@@ -52,6 +52,20 @@ else
   exit 1
 fi
 
+# Set up tmp dir in case it's not configured
+if [ -z "${GRAVITY_TMPDIR}" ]; then
+  GRAVITY_TMPDIR="/tmp"
```

Comment:
```suggestion
: "${GRAVITY_TMPDIR:=/tmp}"
```

See https://github.com/pi-hole/pi-hole/blob/d86b325dfe4d39b7bc523672bb22f1bdbd12265f/automated%20install/basic-install.sh#L38

```sh
Viking:~$ : "${RED:=blue}"
Viking:~$ echo $RED
blue
Viking:~$ unset RED
Viking:~$ RED='green'
Viking:~$ : "${RED:=blue}"
Viking:~$ echo $RED
green
```